test/zXP.test.ts

```diff
@@ -138,6 +140,8 @@ describe("ZXP", () => {
             await mockErc721.connect(staker2)["safeTransferFrom(address,address,uint256)"](s2, gameVault.address, s2nft);
         });
         it("Gets season registry", async () => {
+            // shouldnt I be able to get this info through the gameRegistry?
+            // why do we have another registry?

```

Comment:
These are the ObjectRegistrys stored in the Seasons struct.
It is a separate set of contracts stored and managed separately from the gameRegistrys set of contracts. 

We're able to make different rules on this set of contracts for how they can be updated and added.